From 0aa12762765dedb4b533d986f651a79ef3660328 Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Sun, 20 Oct 2013 10:11:43 +0200
Subject: [PATCH 1103/1825] a38x: adjust defconfigs in order to enable
 silicone initial support

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit bf4c68a3a0093a084e36eeab80c35827a4a0b541

	- disable PP2, XOR, SDIO, USB and SATA

Change-Id: Ifaa651707516f47c3882d2cdc8129cd6ed677317
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Signed-off-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/3760
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/configs/armada_380_v7up_defconfig  |    8 ++++----
 arch/arm/configs/armada_385_v7smp_defconfig |    8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/configs/armada_380_v7up_defconfig b/arch/arm/configs/armada_380_v7up_defconfig
index be5de07..93da650 100644
--- a/arch/arm/configs/armada_380_v7up_defconfig
+++ b/arch/arm/configs/armada_380_v7up_defconfig
@@ -14,11 +14,12 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_ARCH_ARMADA38X=y
 # CONFIG_MV_INCLUDE_CESA is not set
 # CONFIG_MV_INCLUDE_TDM is not set
-CONFIG_MV_INCLUDE_SPI=y
 CONFIG_MV_ETH_TXQ=8
 # CONFIG_NET_SKB_RECYCLE is not set
-CONFIG_MV_ETH_PP2=y
-CONFIG_MV_INCLUDE_ETH_PHY=y
+# CONFIG_MV_INCLUDE_USB is not set
+# CONFIG_MV_INCLUDE_XOR is not set
+# CONFIG_MV_INCLUDE_INTEG_SATA is not set
+# CONFIG_MV_INCLUDE_SDIO is not set
 CONFIG_MV_ETH_TXQ_DESC=1024
 CONFIG_MV_ETH_DEBUG_CODE=y
 CONFIG_MV_ETH_STAT_DBG=y
@@ -133,7 +134,6 @@ CONFIG_USB_STORAGE_SDDR09=y
 CONFIG_USB_STORAGE_SDDR55=y
 CONFIG_USB_STORAGE_JUMPSHOT=y
 CONFIG_MMC=y
-CONFIG_MMC_MVSDIO=y
 CONFIG_NEW_LEDS=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_MV=y
diff --git a/arch/arm/configs/armada_385_v7smp_defconfig b/arch/arm/configs/armada_385_v7smp_defconfig
index 7a09eef..dbba244 100644
--- a/arch/arm/configs/armada_385_v7smp_defconfig
+++ b/arch/arm/configs/armada_385_v7smp_defconfig
@@ -14,11 +14,12 @@ CONFIG_MODULE_UNLOAD=y
 CONFIG_ARCH_ARMADA38X=y
 # CONFIG_MV_INCLUDE_CESA is not set
 # CONFIG_MV_INCLUDE_TDM is not set
-CONFIG_MV_INCLUDE_SPI=y
 CONFIG_MV_ETH_TXQ=8
 # CONFIG_NET_SKB_RECYCLE is not set
-CONFIG_MV_ETH_PP2=y
-CONFIG_MV_INCLUDE_ETH_PHY=y
+# CONFIG_MV_INCLUDE_USB is not set
+# CONFIG_MV_INCLUDE_XOR is not set
+# CONFIG_MV_INCLUDE_INTEG_SATA is not set
+# CONFIG_MV_INCLUDE_SDIO is not set
 CONFIG_MV_ETH_TXQ_DESC=1024
 CONFIG_MV_ETH_DEBUG_CODE=y
 CONFIG_MV_ETH_STAT_DBG=y
@@ -136,7 +137,6 @@ CONFIG_USB_STORAGE_SDDR09=y
 CONFIG_USB_STORAGE_SDDR55=y
 CONFIG_USB_STORAGE_JUMPSHOT=y
 CONFIG_MMC=y
-CONFIG_MMC_MVSDIO=y
 CONFIG_NEW_LEDS=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_MV=y
-- 
1.7.5.4

